validator/testcases/regex.py

```diff
@@ -2901,9 +2915,34 @@ def munge_filename(name):
                         "the `%s` preference branch" % branch)))
         for branch, reason in BANNED_PREF_BRANCHES))
 
-STRING_REGEXPS += PREF_REGEXPS
+# For tests in literal strings, add help text suggesting passing the
+# preference directly to reference getter functions.
+PREF_STRING_HELP = (
+    "If you are reading, but not writing, this preference, please consider "
+    "passing a string literal directly to `Preferences.get()` or "
+    "`nsIPrefBranch.get*Pref`.")
+
+
+def maybe_tuple(value):
```

Comment:
How about naming this `to_tuple`? Also, wouldn't be against a docstring (even if the code is easy to understand)